src/index.js

```diff
@@ -104,17 +102,17 @@ export default function supernova(env) {
         constraints,
         direction,
         selectionsAPI.isModal(),
-        theme.name(),
-        theme.table.backgroundColor,
+        theme,
         keyboard.active,
-        translator.language(),
         rect.width,
+        announce,
+        changeSortOrder,
       ]);
 
       // this is the one we want to use for carbon
       useEffect(() => {
-        renderWithCarbon({ env, translator, rootElement, model, theme, selectionsAPI, app, rect, layout });
-      }, [layout, model, selectionsAPI.isModal(), theme.name(), manageData, translator.language(), app]);
+        renderWithCarbon({ env, rootElement, model, theme, selectionsAPI, app, rect, layout });
+      }, [layout, model, selectionsAPI.isModal(), theme, translator.language(), app, changeSortOrder]);
```

Comment:
@vcellu Why is managedData passed to the native render? It is a pure function, you can just import it where needed?